monifu/shared/src/main/scala/monifu/reactive/internals/operators/debounce.scala

```diff
@@ -45,7 +45,7 @@ private[reactive] object debounce {
         private[this] var hasValue = false
 
         locally {
-          scheduleNext(timeoutMillis)
+          scheduleNext(initialDelay.toMillis)
```

Comment:
You're making a mistake in thinking that this operator after that timespan of silence it emits items (plural). After that delay it emits a single item. There's is no `initialDelay` because there is a single `delay` value we are dealing with.